packages/octasine/PKGBUILD

```diff
@@ -1,15 +1,16 @@
 # Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
 # Contributor: Christopher Arndt <aur -at- chrisarndt -dot- de>
 # Contributor: Mathias Buhr <napcode@aparatus.de>
+# Contributor: Florian Hülsmann <fh@cbix.de>
 
 _name=OctaSine
 pkgname=${_name,,}
 pkgver=0.9.1
-pkgrel=1
+pkgrel=2
 pkgdesc='A four-operator stereo FM synthesizer VST2 plugin'
```

Comment:
We should update the description to either add 'CLAP' or remove 'VST2'.

Reply:
9708cc8fa40ef3f0eb75faa6b038ec25d384d7b5